### ISSUE TITLE
Paper Spigot + multiverse

### ISSUE THREAD

**Issue by zetyu** [Multiverse/Multiverse-Core#1681]
Hello guys, 
I'm using paper spigot and when I have the plugin installed it adds prefix in front of my players name and in the config I can turn the prefixchatformat to false but when I want to apply the changes it turns back to the default config. I don't know if it is because i'm running on paper spigot but it seems to be related to that because on another server I own it works perfectly. 
If you have any idea it would be cool.
Thanks.

**Comment by TWSSYesterday** [comment]
Try /mv conf prefixchat false

**Comment by zetyu** [comment]
thanks a lot men, it worked perfectly !

**Comment by TWSSYesterday** [comment]
Config file isn't saving properly, well known issue, perhaps MV3 will fix this.

**Comment by FernFerret** [comment]
Just to chime in on this, it isn't a bug. When the plugin is closed, the config is overwritten to ensure that any additional changes that were made ingame were sync'd to disk.

This is why we *heavily stress* the use of the `/mv conf` commands over manual config editing. If you want to edit configs by hand there is large possibility for error.

Back in my day (read: 4 years ago) people kept editing YAML configs and messing them up, so I gave people a way to do all editing in game.

tldr; don't edit configs by hand unless the server is stopped, use `/mv conf`
